src/NeoSharp.Core/VM/IStackAccessor.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections;
+using System.Numerics;
+
+namespace NeoSharp.Core.VM
+{
+    public interface IStackAccessor
+    {
```

Comment:
Is this interface needed? Then should be moved to the `NeoSharp.VM.Interop` package

Reply:
I've realized it was probably not the best thing to introduce.